src/main/java/dev/latvian/mods/tanky/client/TankControllerRenderer.java

```diff
@@ -36,15 +34,15 @@ public void render(TankControllerBlockEntity entity, float delta, PoseStack matr
 		int light = LevelRenderer.getLightColor(entity.getLevel(), entity.tank.getFluid().getFluid().defaultFluidState().createLegacyBlock(), entity.getBlockPos().above());
 
 		FluidStack fluid = entity.tank.getFluid();
-		var renderProperties = RenderProperties.get(fluid.getFluid());
+		var renderProperties = ((IClientFluidTypeExtensions)fluid.getFluid().getFluidType().getRenderPropertiesInternal());
```

Comment:
Couldn't this be `IClientFluidTypeExtensions.of` instead?

Reply:
Didn't know it was an optional. Can fix

Reply:
Corrected with [c981f35](https://github.com/latvian-dev/Tanky/pull/2/commits/c981f35a213c36d9fff25a18f3199993a5af39ac)